logical/system_view.go

```diff
@@ -61,6 +61,9 @@ type SystemView interface {
 	// EntityInfo returns a subset of information related to the identity entity
 	// for the given entity id
 	EntityInfo(entityID string) (*Entity, error)
+
+	// PluginEnv returns Vault environment information used by plugins
+	PluginEnv() (*PluginEnvironment, error)
```

Comment:
Should probably have this accept a context as a parameter instead of using the Background context for gRPC

Reply:
Added.